spec/spec_helper.rb

```diff
@@ -102,3 +102,11 @@
   Kernel.srand config.seed
 =end
 end
+
+def params_wrapper(args)
+  if defined?(::Rails) && Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new('5.0.0.beta1')
+    { params: args }
+  else
+    args
+  end
+end
```

Comment:
Please add a comment explaining the change to ".. keyword arguments syntax in ActionController::TestCase and ActionDispatch::Integration HTTP request methods." and include a link to the ActionPack changelog (https://github.com/tgxworld/rails/blob/master/actionpack/CHANGELOG.md)